### ISSUE TITLE
can't remove the repo in the browser

### ISSUE THREAD

**Issue by fengzhao** [goharbor/harbor#6289]
**Expected behavior and actual behavior:**

remove the repo

**Steps to reproduce the problem:**
1.  i delete the repo in the broswer 
2. run the cammond  in the host 
``` shelll
docker run -it --name gc --rm --volumes-from registry   goharbor/registry-photon:v2.6.2-v1.6.0  garbage-collect /etc/registry/config.yml
```
3. but after i run the 
``` shell
docker-compose start
```
4. the browse didn't remove the repo 。and i can't remove the record in the browser  anymore 。

![image](https://user-images.githubusercontent.com/6206340/48546627-c8553080-e903-11e8-8fd5-1b5199b28fce.png)
![image](https://user-images.githubusercontent.com/6206340/48546674-dd31c400-e903-11e8-891b-1c37ae3c953d.png)

how can i really remove the repo in the browser.

**Versions:**
Please specify the versions of following systems.
- harbor version: [1.6.0]
- docker engine version: [18.09]
- docker-compose version: [1.18.0]

**Additional context:**
* **Harbor config files:** You can get them by packaging `harbor.cfg` and files in the same directory, including subdirectory.
* **Log files:** You can get them by package the `/var/log/harbor/` .

**Comment by wy65701436** [comment]
could you refer to the workaround of this issue? https://github.com/goharbor/harbor/issues/5078

**Comment by fengzhao** [comment]
> could you refer to the workaround of this issue? #5078
ok , i got it !
due to the bug of  the docker registry v2.6.2(cache invalidation),  so the harbor can get 200 from API when to delete the repository and cause the strange behaviour.   and the docker registry will solve it on  2.7  .  but t no mention the schedule  ,  that's why harbor can't solve it .

but the docker  has release registry 2.7.0-rc.0 .  can harbor vertify wheather the docker  has solved the problem in  registry-2.7.0-rc.0 version really and  solve my problem in the next  harbor pre-release version?

thanks a lot!!!

**Comment by wy65701436** [comment]
@fengzhao, we have a solution to resolve the cache invalidation issue -- it already merged into master branch -- that is to clean cache in the GC job execution. 

The online GC will be ready in harbor v1.7.0, or you can try with the latest master code.
